src/tests/dbus-tests/test_70_encrypted.py

```diff
@@ -5,12 +5,16 @@
 import tempfile
 import time
 import unittest
+import configparser
 
 from distutils.version import LooseVersion
 
 import udiskstestcase
 
 
+UDISKS_CONFIG_FILE = "/etc/udisks2/udisks2.conf"
```

Comment:
Shouldn't this be read from the sources instead?

Reply:
Depending on the "uninstalled" vs. system case...

Reply:
We already "install" some config and policy files for the tests, I have added new commit that adds `udisks2.conf` too.